profile-widget/tankitem.cpp

```diff
@@ -111,6 +110,4 @@ void TankItem::replot()
 void TankItem::setHorizontalAxis(DiveCartesianAxis *horizontal)
 {
 	hAxis = horizontal;
-	connect(hAxis, SIGNAL(sizeChanged()), this, SLOT(replot()));
```

Comment:
IIRC when you use profile edit functionality (right click on the profile) and drag the final handle around so that the dive gets longer, that will change the hAxis and that in return should create this signal. But I didn't try this.

Reply:
Thanks for pointing this out - I will test this tomorrow.

Reply:
OK, I'll leave it sitting here then :-)

Reply:
btw, did you have a chance to look at this question? As usual, it's entirely possible that I'm wrong...

Reply:
Note that I didn't look at this recently, I just pushed old commits that were lying around.

But when I looked at it a few days ago, the axis resized properly when moving the last marker.

Reply:
that's encouraging.
I could test this some more and move towards merging (after fixing commit messages) - or is this something that you are still actively working on?
It's so hard to see from here where you are currently spending your energy :-)

Reply:
Feel free to commit if nothing breaks.